pandas/tests/indexing/interval/test_interval_new.py

```diff
@@ -8,89 +8,65 @@
 
 
 class TestIntervalIndex:
-    def setup_method(self, method):
-        self.s = Series(np.arange(5), IntervalIndex.from_breaks(np.arange(6)))
+    @pytest.fixture
```

Comment:
we should prob remove one of these files, this has been 'new' for a while

Reply:
agreed